crates/libeq_wld/src/parser/fragments/material.rs

```diff
@@ -93,14 +30,18 @@ pub struct MaterialFragment {
     // Bit 3 ........ Set to 1 if the texture is masked and semi-transparent.
     // Bit 4 ........ Set to 1 if the texture is masked but not semi-transparent.
     // Bit 31 ...... Apparently must be 1 if the texture isn’t transparent.
-    pub transparency_flags: TransparencyFlags,
+    pub render_method: RenderMethod,
 
     /// This typically contains 0x004E4E4E but has also bee known to contain 0xB2B2B2.
     /// Could this be an RGB reflectivity value?
-    pub params2: u32,
+    /// RGBPEN %d, %d, %d
+    pub rgb_pen: u32,
```

Comment:
Again, this matches with the ASCII material defs. One thing I'm unsure of here is exactly what this _is_. It may just be gaps in my graphics knowledge. Any idea what `pen` would mean in this context?

Reply:
This is why I'm excited to get these frags into Godot, as I intend to make it possible to look at meshes in the game and see their full fragment information - then I can try to visually correlate these values to the actual mesh as displayed in the game